consensus/hotstuff/committees/consensus_committee_test.go

```diff
@@ -740,26 +740,30 @@ func TestRemoveOldEpochs(t *testing.T) {
 }
 
 // addExtension adds the extension to the mocked epoch, by updating its final view.
-func addExtension(epoch *protocolmock.Epoch, ext flow.EpochExtension) {
+func addExtension(epoch *protocolmock.CommittedEpoch, ext flow.EpochExtension) {
 	epoch.On("FinalView").Unset()
 	epoch.On("FinalView").Return(ext.FinalView, nil)
 }
 
-// newMockEpoch returns a new mocked epoch with the given fields
-func newMockEpoch(counter uint64, identities flow.IdentityList, firstView uint64, finalView uint64, committed bool) *protocolmock.Epoch {
+// newMockCommittedEpoch returns a new mocked committed epoch with the given fields
+func newMockCommittedEpoch(counter uint64, identities flow.IdentityList, firstView uint64, finalView uint64) *protocolmock.CommittedEpoch {
 
-	epoch := new(protocolmock.Epoch)
+	epoch := new(protocolmock.CommittedEpoch)
 	epoch.On("Counter").Return(counter, nil)
 	epoch.On("RandomSource").Return(unittest.RandomBytes(32), nil)
 	epoch.On("InitialIdentities").Return(identities.ToSkeleton(), nil)
 	epoch.On("FirstView").Return(firstView, nil)
 	epoch.On("FinalView").Return(finalView, nil)
-	if committed {
-		// return nil error to indicate the epoch is committed
-		epoch.On("DKG").Return(nil, nil)
-	} else {
-		epoch.On("DKG").Return(nil, protocol.ErrNextEpochNotCommitted)
-	}
+	// return nil error to indicate the epoch is committed
```

Comment:
```suggestion
```
This comment is a bit misleading now, because we are only dealing with committed epochs.